assets/js/blocks/cart-checkout/cart-i2/edit.tsx

```diff
@@ -162,6 +158,10 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
 	);
 };
 
+const ALLOWED_BLOCKS: string[] = [
+	'woocommerce/cart-items-block',
```

Comment:
I saw your note in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4781/files r/e naming. If you wanted to reserve this name for actual items, you could name this something like "cart-item-container-block". If easier.